observability/prometheus/values.yaml

```diff
@@ -0,0 +1,318 @@
+    ## Example which selects PodMonitors with label "prometheus" set to "somelabel"
+    # podMonitorSelector:
+    #   matchLabels:
+    #     prometheus: somelabel
+
+    ## If nil, select own namespace. Namespaces to be selected for PodMonitor discovery.
+    podMonitorNamespaceSelector: {}
+    ## Example which selects PodMonitor in namespaces with label "prometheus" set to "somelabel"
+    # podMonitorNamespaceSelector:
+    #   matchLabels:
+    #     prometheus: somelabel
+
+    ## How long to retain metrics
+    ##
+    retention: 1d
+    ## Maximum size of metrics
+    ##
+    retentionSize: ""
+    ## Log level for Prometheus be configured in
+    ##
+    logLevel: info
+    ## Log format for Prometheus be configured in
+    ##
+    logFormat: logfmt
+    ## Standard object's metadata. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#metadata
+    ## Metadata Labels and Annotations gets propagated to the prometheus pods.
+    ##
+    podMetadata:
+      labels:
+        azure.workload.identity/use: "true"
+    #   app: prometheus
+    #   k8s-app: prometheus
+
+    ## Pod anti-affinity can prevent the scheduler from placing Prometheus replicas on the same node.
+    ## The default value "soft" means that the scheduler should *prefer* to not schedule two replica pods onto the same node but no guarantee is provided.
+    ## The value "hard" means that the scheduler is *required* to not schedule two replica pods onto the same node.
+    ## The value "" will disable pod anti-affinity so that no anti-affinity rules will be configured.
+    podAntiAffinity: "hard"
+    ## If anti-affinity is enabled sets the topologyKey to use for anti-affinity.
+    ## This can be changed to, for example, failure-domain.beta.kubernetes.io/zone
+    ##
+    podAntiAffinityTopologyKey: kubernetes.io/hostname
+    ## Assign custom affinity rules to the prometheus instance
+    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+    ##
+    affinity: {}
+    # nodeAffinity:
+    #   requiredDuringSchedulingIgnoredDuringExecution:
+    #     nodeSelectorTerms:
+    #     - matchExpressions:
+    #       - key: kubernetes.io/e2e-az-name
+    #         operator: In
+    #         values:
+    #         - e2e-az1
+    #         - e2e-az2
+
+    ## The remote_write spec configuration for Prometheus.
+    ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api-reference/api.md#remotewritespec
+    remoteWrite:
+    - url: "http://localhost:8081/api/v1/write"
+      queueConfig:
+        capacity: 2500
+        maxShards: 500
+        minShards: 1
+        maxSamplesPerSend: 2000
+        batchSendDeadline: 60s
+        minBackoff: 30ms
+        maxBackoff: 256s
+      metadataConfig:
+        send: false
+      writeRelabelConfigs:
+      - sourceLabels: [__name__, source]
+        action: keep
+        separator: ;
+        regex: ".*"
```

Comment:
I imagine that it's a placeholder for future filtering since the current expression forwards all metrics?

Reply:
I removed it.  We can add it back later if we need to filter further.